### ISSUE TITLE
Possible bug in batchnorm call

### ISSUE THREAD

**Issue by francoisgermain** [wilkinghoff/ssl4asd#1]
Thanks for sharing the code.

I was wondering if there might be a bug in this line of the code:
https://github.com/wilkinghoff/ssl4asd/blob/239f80ace7190397eb91ae8c25a1111e643aea4e/main_statex%2Bfeatex.py#L175

I couldn't find the details needed to confirm in the ICASSP paper, but if I compare with [the IJCNN model](https://github.com/wilkinghoff/sub-cluster-AdaCos/blob/260d509c2bcaed255a749b0fa0b306f2d8129657/main.py#L113) and the typical ResNet architecture, it seems like it should be:
`xr = tf.keras.layers.BatchNormalization()(xr)`

Note: the same line appears in [the ICASSP 2023 repo](https://github.com/wilkinghoff/icassp2023/blob/27dcf3ab0182ade1eebf364db3bb2bcfd70bc430/main.py#L121) so I'll post an issue there.

I'll put in a pull request with the fix shortly if I'm not beaten to it. Thanks!

**Comment by francoisgermain** [comment]
Associated pull request: https://github.com/wilkinghoff/ssl4asd/pull/3

**Comment by wilkinghoff** [comment]
Good catch, you are absolutely right, thank you!
